packages/mask/src/plugins/RedPacket/SNSAdaptor/RedPacketNft.tsx

```diff
@@ -397,6 +398,10 @@ export function RedPacketNft({ payload }: RedPacketNftProps) {
                                     erc721TokenDetailed?.info.mediaUrl ? '' : classes.loadingTokenImg,
                                 )}
                                 src={erc721TokenDetailed?.info.mediaUrl ?? previewNftImg}
+                                onError={(event) => {
```

Comment:
That component also could be used here. If the initial source fails to load, it presents a preview image.